OpenRA.Mods.Common/ColorValidator.cs

```diff
@@ -86,6 +86,62 @@ public bool IsValid(Color askedColor, out Color forbiddenColor, IEnumerable<Colo
 			return true;
 		}
 
+		public List<Color> GetModPresetColors()
```

Comment:
It would be better to stick with `HSLColor`s from the start, otherwise you need to needlessly convert between types.